### PR TITLE
chore: add cfn-nag to all yaml changes

### PR DESCRIPTION
**Pull request by rsmayda**
NOTE: All code was deployed and tested

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloudformation/bulkExport.yaml [hunk 1]
```diff
@@ -6,6 +6,11 @@
 Resources:
   GlueJobRelatedLambdaRole:
     Type: AWS::IAM::Role
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W11
+            reason: '* only the the glue jobs and is required due to glue requirements'
```
Comment (nguyen102) [inline cloudformation/bulkExport.yaml]: This reason is confusing. Can we rephrase it?

File: serverless.yaml [hunk 2]
```diff
@@ -569,6 +612,12 @@ resources:
           KmsKeyId: !GetAtt LogKMSKey.Arn
         DependsOn:
           - LogKMSKey
+      ApiGatewayMethodMetadataGet:
+        Metadata:
+          cfn_nag:
+            rules_to_suppress:
+              - id: W45
+                reason: 'This API endpoint should not have authentication required (due to the FHIR spec)'
```
Comment (nguyen102) [inline serverless.yaml]: rephrase: `should not require authentication`

**Review by nguyen102 — COMMENTED** [review]
Some rephrasing of cfn-nag reasons. Otherwise looks good.

File: serverless.yaml (hunk 2 above)
Comment (rsmayda) [reply serverless.yaml]:
```suggestion
                reason: 'This API endpoint should not require authentication (due to the FHIR spec)'

```

File: cloudformation/bulkExport.yaml (hunk 1 above)
Comment (rsmayda) [reply cloudformation/bulkExport.yaml]: updated

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**